static/dailyjobs.html

```diff
@@ -0,0 +1,165 @@
+<html>
+<head>
+  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+  <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
+  <script src="js/flot/jquery.flot.min.js"></script>
+  <script src="js/flot/jquery.flot.time.min.js"></script>
+  <script src="js/flot/jquery.flot.selection.min.js"></script>
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+  <title>
+    Tracking SETA
+  </title>
+</head>
+<body>
+  <h1>Tracking SETA</h1>
+  <p>
+    <a href="http://alertmanager.allizom.org/data/dailyjobs/">Raw Data</a><br />
+    <p>
+      Start Date: <input type="text" id="datepicker1">
+      End Date: <input type="text" id="datepicker2">
+      Branch: <select id="branch" onchange="changeBranch();">
+                <option id="mozilla-inbound"> mozilla-inbound </option>
+                <option id="fx-team"> fx-team </option>
+              </select>
+      Platform: <select id="platform" onchange="changePlatform();">
+                <option id="all"> All </option>
+                <option id="linux"> Linux </option>
+                <option id="osx"> OSX </option>
+                <option id="windows"> Windows </option>
+                <option id="windows"> Android </option>
+                </select>
+    </p>
+  <div id="placeholder" style="width:1200px;height:600px"></div>
+  <span id="hoverdata"></span>
+  <span id="clickdata"></span>
+  <script>
+    $(function() {
+      $( "#datepicker1" ).datepicker({
+        dateFormat: 'yy-mm-dd',
+        onSelect: function(selected, event){
+          callStartDate(selected);
+          }
+        });
+    });
+
+    $(function() {
+      $( "#datepicker2" ).datepicker({
+        dateFormat: 'yy-mm-dd',
+        onSelect: function(selected, event){
+          callEndDate(selected);
+        }
+      });
+    });
+
+    var start_date;
+    var end_date;
+    var branch = document.getElementById("branch").value;
+    var platform = document.getElementById("platform").value;
+
+    function changeBranch(){
+      branch = document.getElementById("branch").value;
+      callJson(start_date, end_date);
+    }
+
+    function changePlatform(){
+      platform = document.getElementById("platform").value;
+      callJson(start_date, end_date);
+    }
+
+    function callStartDate(date){
+      start_date = date;
+      callJson(start_date, end_date);
+    }
+
+    function callEndDate(date){
+      end_date = date;
+      callJson(start_date, end_date);
+    }
+
+    function callJson(start_date, end_date){
```

Comment:
This should be named something more specific than callJson, maybe getDailyJobsByDate?